src/app/modules/core/manager/landlord/components/view/manager-landlord-view.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, OnInit } from '@angular/core'
```

Comment:
Please, rename the file, the component class, all the related files and the selector in `landlord-view` instead of `manager-landlord-new`. Most probably this component will be shared with the landlord module. Thanks.